### PR TITLE
[otci] dns query timeout fix

### PR DESCRIPTION
**Pull request by priyankaCh22**
Hi,
We discovered that `dns resolve` takes a little longer i.e. 15sec to finish (rather than default timeout of 10s). Thus using a larger timeout for dns resolve (and other dns commands), e.g. 30s has helped.
Thanks

### CODE REVIEW

File: tools/otci/otci/otci.py [hunk 1]
```diff
@@ -99,6 +99,9 @@ def execute_command(self, cmd: str, timeout: float = 10, silent: bool = False) -
         if not silent:
             self.__logger and self.__logger.info('> %s', cmd)
 
+        if "dns" in cmd:
```
Comment (simonlingoogle) [inline tools/otci/otci/otci.py:102]: Can we use a different timeout in methods below?
- dns_browse
- dns_resolve
- dns_resolve_service

It seems a little hardcoding to treat `dns` commands differently in `execute_command`?

Comment (priyankaCh22) [reply tools/otci/otci/otci.py:102]: Sure will do that way.

**Comment by simonlingoogle** [comment]
@abtink Any suggestion on the proper timeout for `dns resolve`, which is a complex procedure involving both AAAA and A queries.

**Review by simonlingoogle — APPROVED** [review]
LGTM :+1:

**Review by abtink — APPROVED** [review]
LGTM. Thanks. 👍

**Review by jwhui — APPROVED** [review]

**Pull request merged**